src/init.php

```diff
@@ -33,6 +33,7 @@
 Buddy::setVersionFile(__DIR__ . '/../APP_VERSION');
 
 $opts = CliArgsProcessor::run();
+$authToken = getenv('BUDDY_TOKEN') ?: null;
```

Comment:
As discussed, consider dropping the env. var. here.

Reply:
There is nothing in /proc/[pid]/environ file, but still added putenv to rewrite variable